consensus/babe/babe_test.go

```diff
@@ -508,22 +509,27 @@ func TestBuildBlock(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// the hash of an empty trie
-	emptyRootHash, err := common.HexToHash("0x03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c111314")
+	// hash of parent header
+	parentHash, err := common.HexToHash("0x01d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da2")
```

Comment:
Where are the example hashes coming from?

Reply:
we are kinda assuming the runtime is creating the correct hash for `stateRoot` and `extrinsicsRoot`, and the parent hash is the hash of the parent block